airflow/models.py

```diff
@@ -3267,6 +3265,61 @@ def clear(
         session.close()
         return count
 
+    @classmethod
+    def clear_dags(
+        cls, dags,
+        start_date=None,
+        end_date=None,
+        only_failed=False,
+        only_running=False,
+        confirm_prompt=False,
+        include_subdags=True,
+        reset_dag_runs=True,
+        dry_run=False):
+        all_tis = []
+        for dag in dags:
+            tis = dag.clear(
+                start_date=start_date,
+                end_date=end_date,
+                only_failed=only_failed,
+                only_running=only_running,
+                confirm_prompt=False,
+                include_subdags=include_subdags,
+                reset_dag_runs=reset_dag_runs,
+                dry_run=True)
+            all_tis.extend(tis)
+
+        if dry_run:
+            return all_tis
+
+        count = len(all_tis)
+        do_it = True
+        if count == 0:
+            print("Nothing to clear.")
+            return 0
+        if confirm_prompt:
+            ti_list = "\n".join([str(t) for t in all_tis])
```

Comment:
landscape isn't sophisticated enough to deal with locals(), so it thinks it's not used:
```ti_list = "\n".join([str(t) for t in tis])
question = (
    "You are about to delete these {count} tasks:\n"
    "{ti_list}\n\n"
    "Are you sure? (yes/no): ").format(**locals())```